node/src/reactor.rs

```diff
@@ -58,7 +58,6 @@ use tokio::time::{Duration, Instant};
 use tracing::{debug, debug_span, error, info, instrument, trace, warn, Span};
 use tracing_futures::Instrument;
 
-#[cfg(target_os = "linux")]
```

Comment:
Removed by mistake?

Reply:
Intentional, actually, as currently build fails on macOS

Reply:
Tried my best to revert as closely as possible here 6af61de22e0b31f0dd186215e2a07cf30aa55596